src/test_client.cc

```diff
@@ -37,6 +37,10 @@ static void add_test() {
     LOG_INFO(TEST,  a << " + " << b << " = " << result);
 }
 
+TEST(functional, HandlesPositiveInput) {
```

Comment:
What is this doing?

Reply:
Good catch!  That was residual code when I was playing around with GoogleTest.  Removed.